tasks/deploy.yml

```diff
@@ -14,7 +14,9 @@
     version: "{{ drupal_deploy_version }}"
     update: "{{ drupal_deploy_update }}"
     force: true
+    depth: "{{ drupal_deploy_depth }}"
     dest: "{{ drupal_deploy_dir }}"
+    separate_git_dir: "{{ drupal_deploy_gitdir }}"
```

Comment:
Note that this is a pretty recent addition to Ansible (2.7), so we will also need to update the minimum ansible version in meta/main.yml